Robin/HW1_Robin/mock.css

```diff
@@ -0,0 +1,72 @@
+body{
+	background:url("asset/page-mock.png") no-repeat;
+	background-size: 1024px 885px;
+	font-family: Verdana, sans-serif;
+
+}
+
+h1{
+	color: #666666;
+	font-weight: normal;
+	padding-top: 12px;
+	margin-left: 260px;
+	font-size:47px;
+	letter-spacing: 0.5px;
```

Comment:
usually, we group related style together, color font-weight font size letter-spacing should come together. per box model margin and then padding